projects/m1/006-sum-of-digits-in-a-integer/solutions/DomenicoTraina/sum-of-digits-in-a-integer.py

```diff
@@ -0,0 +1,11 @@
+
+number= int (input ("Please enter a 4 digit number: "))
+
+number_string = str(number)
+digit1=int(number_string[0])
+digit2=int(number_string[1])
+digit3=int(number_string[2])
+digit4=int(number_string[3])
+
+sum_of_digits=digit1+digit2+digit3+digit4
```

Comment:
Alright, that is true until the number is exactly of four digits.

But if it will be a number of 1000 digits?